Errors on installation

### Instrument

_No response_

### What happened?

Tried installing on a clean conda environments with 

git clone -b v0.3 https://github.com/kevin218/Eureka.git
cd Eureka
pip install -e '.[jwst]'

getting the following, very long series of warnings/errors:

after that, if I try to run run_eureka.py, it fails with "No module named Eureka"

### Error traceback output

[error.txt](https://github.com/kevin218/Eureka/files/9152704/error.txt)


### What operating system are you using?

Mac OSx

### What version of Python are you running?

3.9.7

### What Python packages do you have installed?

i am trying to install in a brand new conda environment

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

I have also tried this on a Rosetta2 terminal, but I keep getting a massive
error when it hits h5py. conda installing h5py doesn't seem to do anything.

(eureka) ***@***.*** Eureka % pip install -e '.[jwst]'

Obtaining file:///Users/rasmussen/Eureka/Eureka

  Installing build dependencies ... done

  Checking if build backend supports build_editable ... done

  Getting requirements to build wheel ... done

  Preparing metadata (pyproject.toml) ... done

Collecting astraeus@ ***@***.***

  Cloning https://github.com/kevin218/Astraeus (to revision main) to
/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-56llqnph/astraeus_45350510b1744b179336cb617dbfc87c

  Running command git clone --filter=blob:none --quiet
https://github.com/kevin218/Astraeus
/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-56llqnph/astraeus_45350510b1744b179336cb617dbfc87c

  Resolved https://github.com/kevin218/Astraeus to commit
20373ac916fbfdcb85a4f23dfd0690a21dc2771f

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Preparing metadata (pyproject.toml) ... done

Collecting astroquery

  Using cached astroquery-0.4.6-py3-none-any.whl (4.5 MB)

Collecting astropy

  Using cached astropy-5.1-cp39-cp39-macosx_11_0_arm64.whl (6.7 MB)

Collecting corner

  Using cached corner-2.2.1-py3-none-any.whl (15 kB)

Collecting batman-package

  Using cached batman_package-2.4.9-cp39-cp39-macosx_11_0_arm64.whl

Collecting crds

  Using cached crds-11.16.6-py3-none-any.whl (1.2 MB)

Collecting pandas

  Using cached pandas-1.4.3-cp39-cp39-macosx_11_0_arm64.whl (10.5 MB)

Collecting h5py<3.2

  Using cached h5py-3.1.0.tar.gz (371 kB)

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... error

  *error*: *subprocess-exited-with-error*



  × pip subprocess to install backend dependencies did not run successfully.

  │ exit code: *1*

  ╰─> [268 lines of output]

      Collecting numpy==1.19.3

        Using cached numpy-1.19.3.zip (7.3 MB)

        Installing build dependencies: started

        Installing build dependencies: finished with status 'done'

        Getting requirements to build wheel: started

        Getting requirements to build wheel: finished with status 'done'

        Preparing metadata (pyproject.toml): started

        Preparing metadata (pyproject.toml): finished with status 'error'

        error: subprocess-exited-with-error



        × Preparing metadata (pyproject.toml) did not run successfully.

        │ exit code: 1

        ╰─> [245 lines of output]

            Running from numpy source directory.

            setup.py:480: UserWarning: Unrecognized setuptools command,
proceeding with generating Cython sources and expanding templates

              run_build = parse_setuppy_commands()

            Processing numpy/random/_bounded_integers.pxd.in

            Processing numpy/random/_philox.pyx

            Processing numpy/random/_bounded_integers.pyx.in

            Processing numpy/random/_sfc64.pyx

            Processing numpy/random/_mt19937.pyx

            Processing numpy/random/bit_generator.pyx

            Processing numpy/random/mtrand.pyx

            Processing numpy/random/_generator.pyx

            Processing numpy/random/_pcg64.pyx

            Processing numpy/random/_common.pyx

            Cythonizing sources

            blas_opt_info:

            blas_mkl_info:

            customize UnixCCompiler

              libraries mkl_rt not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            blis_info:

              libraries blis not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            openblas_info:

            C compiler: clang -Wno-unused-result -Wsign-compare
-Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem
/Users/rasmussen/opt/anaconda3/envs/eureka/include -arch arm64
-I/Users/rasmussen/opt/anaconda3/envs/eureka/include -fPIC -O2 -isystem
/Users/rasmussen/opt/anaconda3/envs/eureka/include -arch arm64



            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var/folders

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var/folders/01

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8

            compile options: '-c'

            clang:
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/source.c

            clang
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/source.o
-L/Users/rasmussen/opt/anaconda3/envs/eureka/lib -lopenblas -o
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/a.out

            ld: warning: ignoring file
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmp3bbx90s8/source.o,
building for macOS-x86_64 but attempting to link with file built for
unknown-arm64

            ld: warning: ignoring file
/Users/rasmussen/opt/anaconda3/envs/eureka/lib/libopenblas.dylib, building
for macOS-x86_64 but attempting to link with file built for macOS-arm64

            Undefined symbols for architecture x86_64:

              "_main", referenced from:

                 implicit entry/start for main executable

            ld: symbol(s) not found for architecture x86_64

            clang: error: linker command failed with exit code 1 (use -v to
see invocation)

              NOT AVAILABLE



            atlas_3_10_blas_threads_info:

            Setting PTATLAS=ATLAS

              libraries tatlas not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            atlas_3_10_blas_info:

              libraries satlas not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            atlas_blas_threads_info:

            Setting PTATLAS=ATLAS

              libraries ptf77blas,ptcblas,atlas not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            atlas_blas_info:

              libraries f77blas,cblas,atlas not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            accelerate_info:

              libraries accelerate not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

            Library accelerate was not found. Ignoring

              libraries veclib not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

            Library veclib was not found. Ignoring

              FOUND:

                extra_compile_args = ['-msse3',
'-I/System/Library/Frameworks/vecLib.framework/Headers']

                extra_link_args = ['-Wl,-framework', '-Wl,Accelerate']

                define_macros = [('NO_ATLAS_INFO', 3), ('HAVE_CBLAS', None)]



              FOUND:

                extra_compile_args = ['-msse3',
'-I/System/Library/Frameworks/vecLib.framework/Headers']

                extra_link_args = ['-Wl,-framework', '-Wl,Accelerate']

                define_macros = [('NO_ATLAS_INFO', 3), ('HAVE_CBLAS', None)]



            non-existing path in 'numpy/distutils': 'site.cfg'

            lapack_opt_info:

            lapack_mkl_info:

              libraries mkl_rt not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            openblas_lapack_info:

            C compiler: clang -Wno-unused-result -Wsign-compare
-Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem
/Users/rasmussen/opt/anaconda3/envs/eureka/include -arch arm64
-I/Users/rasmussen/opt/anaconda3/envs/eureka/include -fPIC -O2 -isystem
/Users/rasmussen/opt/anaconda3/envs/eureka/include -arch arm64



            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var/folders

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var/folders/01

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo

            compile options: '-c'

            clang:
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/source.c

            clang
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/source.o
-L/Users/rasmussen/opt/anaconda3/envs/eureka/lib -lopenblas -o
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/a.out

            ld: warning: ignoring file
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmphzro2mpo/source.o,
building for macOS-x86_64 but attempting to link with file built for
unknown-arm64

            ld: warning: ignoring file
/Users/rasmussen/opt/anaconda3/envs/eureka/lib/libopenblas.dylib, building
for macOS-x86_64 but attempting to link with file built for macOS-arm64

            Undefined symbols for architecture x86_64:

              "_main", referenced from:

                 implicit entry/start for main executable

            ld: symbol(s) not found for architecture x86_64

            clang: error: linker command failed with exit code 1 (use -v to
see invocation)

              NOT AVAILABLE



            openblas_clapack_info:

            C compiler: clang -Wno-unused-result -Wsign-compare
-Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem
/Users/rasmussen/opt/anaconda3/envs/eureka/include -arch arm64
-I/Users/rasmussen/opt/anaconda3/envs/eureka/include -fPIC -O2 -isystem
/Users/rasmussen/opt/anaconda3/envs/eureka/include -arch arm64



            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var/folders

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var/folders/01

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T

            creating
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8

            compile options: '-c'

            clang:
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/source.c

            clang
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/source.o
-L/Users/rasmussen/opt/anaconda3/envs/eureka/lib -lopenblas -llapack -o
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/a.out

            ld: warning: ignoring file
/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptyza7eg8/source.o,
building for macOS-x86_64 but attempting to link with file built for
unknown-arm64

            ld: warning: ignoring file
/Users/rasmussen/opt/anaconda3/envs/eureka/lib/libopenblas.dylib, building
for macOS-x86_64 but attempting to link with file built for macOS-arm64

            ld: warning: ignoring file
/Users/rasmussen/opt/anaconda3/envs/eureka/lib/liblapack.dylib, building
for macOS-x86_64 but attempting to link with file built for macOS-arm64

            Undefined symbols for architecture x86_64:

              "_main", referenced from:

                 implicit entry/start for main executable

            ld: symbol(s) not found for architecture x86_64

            clang: error: linker command failed with exit code 1 (use -v to
see invocation)

              NOT AVAILABLE



            flame_info:

              libraries flame not found in
['/Users/rasmussen/opt/anaconda3/envs/eureka/lib', '/usr/lib']

              NOT AVAILABLE



            atlas_3_10_threads_info:

            Setting PTATLAS=ATLAS

              libraries lapack_atlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries tatlas,tatlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries lapack_atlas not found in /usr/lib

              libraries tatlas,tatlas not found in /usr/lib

            <class 'numpy.distutils.system_info.atlas_3_10_threads_info'>

              NOT AVAILABLE



            atlas_3_10_info:

              libraries lapack_atlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries satlas,satlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries lapack_atlas not found in /usr/lib

              libraries satlas,satlas not found in /usr/lib

            <class 'numpy.distutils.system_info.atlas_3_10_info'>

              NOT AVAILABLE



            atlas_threads_info:

            Setting PTATLAS=ATLAS

              libraries lapack_atlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries ptf77blas,ptcblas,atlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries lapack_atlas not found in /usr/lib

              libraries ptf77blas,ptcblas,atlas not found in /usr/lib

            <class 'numpy.distutils.system_info.atlas_threads_info'>

              NOT AVAILABLE



            atlas_info:

              libraries lapack_atlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries f77blas,cblas,atlas not found in
/Users/rasmussen/opt/anaconda3/envs/eureka/lib

              libraries lapack_atlas not found in /usr/lib

              libraries f77blas,cblas,atlas not found in /usr/lib

            <class 'numpy.distutils.system_info.atlas_info'>

              NOT AVAILABLE



              FOUND:

                extra_compile_args = ['-msse3',
'-I/System/Library/Frameworks/vecLib.framework/Headers']

                extra_link_args = ['-Wl,-framework', '-Wl,Accelerate']

                define_macros = [('NO_ATLAS_INFO', 3), ('HAVE_CBLAS', None)]



            /private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py:275:
UserWarning: Unknown distribution option: 'define_macros'

              warnings.warn(msg)

            running dist_info

            running build_src

            build_src

            building py_modules sources

            creating build

            creating build/src.macosx-11.1-arm64-3.9

            creating build/src.macosx-11.1-arm64-3.9/numpy

            creating build/src.macosx-11.1-arm64-3.9/numpy/distutils

            building library "npymath" sources

            Could not locate executable gfortran

            Could not locate executable f95

            Could not locate executable f90

            Could not locate executable f77

            Could not locate executable xlf90

            Could not locate executable xlf

            Could not locate executable ifort

            Could not locate executable ifc

            Could not locate executable g77

            Could not locate executable g95

            Could not locate executable pgfortran

            don't know how to compile Fortran code on platform 'posix'

            ld: warning: ignoring file _configtest.o, building for
macOS-x86_64 but attempting to link with file built for unknown-arm64

            Undefined symbols for architecture x86_64:

              "_main", referenced from:

                 implicit entry/start for main executable

            ld: symbol(s) not found for architecture x86_64

            clang: error: linker command failed with exit code 1 (use -v to
see invocation)

            Traceback (most recent call last):

              File
"/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptd_1i5ma_in_process.py",
line 363, in <module>

                main()

              File
"/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptd_1i5ma_in_process.py",
line 345, in main

                json_out['return_val'] = hook(**hook_input['kwargs'])

              File
"/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/tmptd_1i5ma_in_process.py",
line 164, in prepare_metadata_for_build_wheel

                return hook(metadata_directory, config_settings)

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/build_meta.py",
line 157, in prepare_metadata_for_build_wheel

                self.run_setup()

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/build_meta.py",
line 248, in run_setup

                super(_BuildMetaLegacyBackend,

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/build_meta.py",
line 142, in run_setup

                exec(compile(code, __file__, 'exec'), locals())

              File "setup.py", line 508, in <module>

                setup_package()

              File "setup.py", line 500, in setup_package

                setup(**metadata)

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-jdcgxmh8/numpy_e8bbb7ea7b1e49c69adc77638b95bd08/numpy/distutils/core.py",
line 169, in setup

                return old_setup(**new_attr)

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/__init__.py",
line 165, in setup

                return distutils.core.setup(**attrs)

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py",
line 148, in setup

                dist.run_commands()

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py",
line 967, in run_commands

                self.run_command(cmd)

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py",
line 986, in run_command

                cmd_obj.run()

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/command/dist_info.py",
line 31, in run

                egg_info.run()

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-jdcgxmh8/numpy_e8bbb7ea7b1e49c69adc77638b95bd08/numpy/distutils/command/egg_info.py",
line 24, in run

                self.run_command("build_src")

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/_distutils/cmd.py",
line 313, in run_command

                self.distribution.run_command(command)

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-build-env-ybo_v3jp/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py",
line 986, in run_command

                cmd_obj.run()

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-jdcgxmh8/numpy_e8bbb7ea7b1e49c69adc77638b95bd08/numpy/distutils/command/build_src.py",
line 144, in run

                self.build_sources()

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-jdcgxmh8/numpy_e8bbb7ea7b1e49c69adc77638b95bd08/numpy/distutils/command/build_src.py",
line 155, in build_sources

                self.build_library_sources(*libname_info)

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-jdcgxmh8/numpy_e8bbb7ea7b1e49c69adc77638b95bd08/numpy/distutils/command/build_src.py",
line 288, in build_library_sources

                sources = self.generate_sources(sources, (lib_name,
build_info))

              File
"/private/var/folders/01/hz_5h9p96qvd1hqn3yjzhh7r0000gn/T/pip-install-jdcgxmh8/numpy_e8bbb7ea7b1e49c69adc77638b95bd08/numpy/distutils/command/build_src.py",
line 378, in generate_sources

                source = func(extension, build_dir)

              File "numpy/core/setup.py", line 663, in get_mathlib_info

                raise RuntimeError("Broken toolchain: cannot link a simple
C program")

            RuntimeError: Broken toolchain: cannot link a simple C program

            [end of output]



        note: This error originates from a subprocess, and is likely not a
problem with pip.

      error: metadata-generation-failed



      × Encountered error while generating package metadata.

      ╰─> See above for output.



      note: This is an issue with the package mentioned above, not pip.

      hint: See above for details.

      [end of output]



  *note*: This error originates from a subprocess, and is likely not a
problem with pip.

*error*: *subprocess-exited-with-error*


× pip subprocess to install backend dependencies did not run successfully.

│ exit code: *1*

╰─> See above for output.


*note*: This error originates from a subprocess, and is likely not a
problem with pip.

(eureka) ***@***.*** Eureka %

On Wed, Jul 20, 2022 at 2:10 PM Kaitlin Rasmussen ***@***.***> wrote:

> I have the M1 chip, does that make a difference?
>
> thanks,
> kaitlin
>
> On Wed, Jul 20, 2022 at 2:08 PM Taylor James Bell <
> ***@***.***> wrote:
>
>> Can you also confirm your Mac has an Intel processor?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/kevin218/Eureka/issues/415#issuecomment-1190759114>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A2E4CUDGNCHZU225OC6KGJ3VVBTDPANCNFSM54EUKJKQ>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

--- a/environment.yml
+++ b/environment.yml
@@ -14,7 +14,6 @@ dependencies:
     - cython
     - dynesty[version='>1.0']
     - emcee[version='>3.0.0']
-    - exotic-ld
     - george
     - gwcs
     - h5py[version='<3.2']
@@ -33,6 +32,7 @@ dependencies:
     - pip:
         - astraeus @ git+https://github.com/kevin218/Astraeus@main#egg=astraeus
         - crds
+        - exotic-ld
         - image_registration @ git+https://github.com/keflavich/image_registration.git@master#egg=image_registration
         - jwst==1.6.0
         - myst-parser # Needed for documentation

